src/index.js

```diff
@@ -1,9 +1,3 @@
-'use strict';
-
-const defaultOptions = {};
-
-function xgboost(options) {
-    options = Object.assign({}, defaultOptions, options);
-}
-
-module.exports = xgboost;
+import loadXGBoost from './loadXGBoost';
+const xgboostWASM = require('../dist/wasm/xgboost');
```

Comment:
seems strange to me have an import and a require in the same file, there's no way to do the require using the import notation?

Reply:
Oh and btw, please .gitignore the dist directory. Files are huge ^^.

Reply:
the problem with the dist is that is needed for the test...

Reply:
let me try something...